Stonith/docs/3nodes/scsi_stonith.md

```diff
@@ -0,0 +1,504 @@
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 22:30:53 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node2: Online
+  node1: Online
+  node3: Offline
+
+(on nodes node3)
+
+-bash-4.2$ sudo pcs cluster status
+[sudo] password for 730727:
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 22:35:38 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node3: Online
+```
+
+# Note: It just hanged here and putty session became inactive. Means the node is rebooted somehow.
+
+```
+After 30/40 seconds node rebooted succesfully & joined cluster again.
+
+on all nodes
+
+-bash-4.2$ sudo pcs cluster  status
+[sudo] password for 730727:
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 23:50:23 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node2: Online
+  node1: Online
+  node3: Online
+```
+
+# Scenario-2:
+
+```
+1. Bring down one node by closing the network using iptables cmds.
+
+Bring down node 10.230.242.95 (node3)
+ip link set down eth0
+
+==> See the results
+
+on node1 and node2:
+
+-bash-4.2$ sudo pcs cluster  status
+[sudo] password for 730727:
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 23:50:23 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node2: Online
+  node1: Online
+  node3: Offline
+
+On node3:
+
+-bash-4.2$ sudo pcs cluster status
+[sudo] password for 730727:
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 22:35:38 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node3: Online
+```
+
+# Note: It just hanged here and putty session became inactive. Means the node is rebooted somehow.
+
+```
+After 30/40 seconds node rebooted succesfully & joined cluster again.
+
+on all nodes
+
+-bash-4.2$ sudo pcs cluster  status
+[sudo] password for 730727:
+Cluster Status:
+ Stack: corosync
+ Current DC: node2 (version 1.1.23-1.el7-9acf116022) - partition with quorum
+ Last updated: Thu Jan  7 23:50:23 2021
+ Last change: Thu Jan  7 03:01:36 2021 by root via cibadmin on node1
+ 3 nodes configured
+ 1 resource instance configured
+
+PCSD Status:
+  node2: Online
+  node1: Online
+  node3: Online
```

Comment:
Add result 
is it pass or failed
like
https://github.com/Seagate/cortx-experiments/blob/main/Stonith/docs/scsi/scsi_stonith.md

Reply:
For VM's testing I haven't taken logs so for now I just added the steps of result. For node hw I have taken out the logs for each stage.